### PR TITLE
Add Census view column definitions

### PR DESCRIPTION
**Pull request by dfsnow**
This PR adds column definitions/documentation to the main ACS/Census view used to populate the model input views. It also adds a few miscellaneous column definitions for other assets.

### CODE REVIEW

File: dbt/models/census/columns.md [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+{% docs column_count_education_associate %}
+Count of all people older than 25 years old with an associate's degree.
+
+ACS variable `B15002_014E` + `B15002_031E`
+{% enddocs %}
+
+## count_education_bachelor
+
+{% docs column_count_education_bachelor %}
+Count of all people older than 25 years old with an bachelor's degree.
```
Comment (jeancochrane) [inline dbt/models/census/columns.md:32]: **[Nitpick]** Small typo:

```suggestion
Count of all people older than 25 years old with a bachelor's degree.
```

File: dbt/models/census/columns.md [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+{% docs column_percent_employment_unemployed %}
+Percent of people 16 years and older unemployed.
+
+ACS variable `B23025_005E` / `B23025_003E`
+{% enddocs %}
+
+## percent_household_family_married
+
+{% docs column_percent_household_family_married %}
+Percent of households that are family, married (married).
```
Comment (jeancochrane) [inline dbt/models/census/columns.md:518]: **[Question, non-blocking]** What's the distinction between the two `married`s here?

File: dbt/models/census/schema.yml [hunk 3]
```diff
@@ -12,169 +12,193 @@ sources:
...
+      - name: count_sex_male
+        description: '{{ doc("column_count_sex_male") }}'
+      - name: count_sex_total
+        description: '{{ doc("column_count_sex_total") }}'
+      - name: geography
+        description: '{{ doc("shared_column_geography") }}'
+      - name: geoid
+        description: '{{ doc("column_geoid") }}'
+      - name: geography
+        description: '{{ doc("shared_column_geography") }}'
```
Comment (jeancochrane) [inline dbt/models/census/schema.yml:118]: **[Nitpick]** Is geography intentionally duplicated here?

```suggestion
      - name: geography
        description: '{{ doc("shared_column_geography") }}'
      - name: geoid
        description: '{{ doc("column_geoid") }}'
```

**Review by wrridgeway — APPROVED** [review]

File: dbt/models/census/columns.md [hunk 4]
```diff
@@ -0,0 +1,757 @@
...
+Count of people (older than 1 year) who have not moved in
+the past 12 months.
+
+ACS variable `B07003_004E`
+{% enddocs %}
+
+## count_mobility_total
+
+{% docs column_count_mobility_total %}
+Count total of migration in the past 12 months.
```
[the remark this answers is not in the document]
Comment (dfsnow) [reply dbt/models/census/columns.md:262]: Migration here is just the Census definition of "anyone who has moved" yeah

File: dbt/models/census/columns.md (hunk 2 above)
Comment (dfsnow) [reply dbt/models/census/columns.md]: Woops, this is a typo!

**Pull request merged**